docs/data/material/getting-started/example-projects/ExampleCollection.js

```diff
@@ -0,0 +1,185 @@
+    src: '/static/images/examples/next.svg',
+  },
+  {
+    name: 'Create React App',
+    label: 'View JS example',
+    tsLabel: 'View TS example',
+    link: 'https://github.com/mui/material-ui/tree/master/examples/material-cra',
+    tsLink:
+      'https://github.com/mui/material-ui/tree/master/examples/material-cra-ts',
+    src: '/static/images/examples/cra.svg',
+  },
+  {
+    name: 'Remix',
+    label: 'View example',
+    link: 'https://github.com/mui/material-ui/tree/master/examples/material-remix-ts',
+    src: '/static/images/examples/remix.svg',
+  },
+  {
+    name: 'Tailwind CSS',
+    label: 'View example',
+    link: 'https://github.com/mui/material-ui/tree/master/examples/material-cra-tailwind-ts',
+    src: '/static/images/examples/tailwindcss.svg',
+  },
+  {
+    name: 'Vite.js',
+    label: 'View JS example',
+    tsLabel: 'View TS example',
+    link: 'https://github.com/mui/material-ui/tree/master/examples/material-vite',
+    src: '/static/images/examples/vite.svg',
+  },
+  {
+    name: 'styled-components',
+    label: 'View JS example',
+    tsLabel: 'View TS example',
+    link: 'https://github.com/mui/material-ui/tree/master/examples/material-cra-styled-components',
+    tsLink:
+      'https://github.com/mui/material-ui/tree/master/examples/material-cra-styled-components-ts',
+    src: '/static/images/examples/styled.png',
+  },
+  {
+    name: 'Gatsby',
+    label: 'View example',
+    link: 'https://github.com/mui/material-ui/tree/master/examples/material-gatsby',
+    src: '/static/images/examples/gatsby.svg',
+  },
+  {
+    name: 'Preact',
+    label: 'View example',
+    link: 'https://github.com/mui/material-ui/tree/master/examples/material-preact',
+    src: '/static/images/examples/preact.svg',
+  },
+  {
+    name: 'CDN',
+    link: 'https://github.com/mui/material-ui/tree/master/examples/material-via-cdn',
+    label: 'View example',
+  },
+  {
+    name: 'Express.js (server-rendered)',
+    label: 'View example',
+    link: 'https://github.com/mui/material-ui/tree/master/examples/material-express-ssr',
+    src: '/static/images/examples/express.png',
+  },
+];
+
+export default function ExampleCollection() {
+  return (
+    <Grid container spacing={2}>
+      {examples.map((example) => (
+        <Grid key={example.name} item xs={12} sm={6}>
+          <Card
+            sx={[
+              (theme) => ({
+                p: 2,
+                display: 'flex',
+                alignItems: 'center',
+                gap: 2,
+                bgcolor: theme.palette.mode === 'dark' ? 'transparent' : '#fff',
+                backgroundImage: 'none',
+                borderRadius: 2,
+                border: '1px solid',
+                borderColor: theme.palette.mode === 'dark' ? '#132F4C' : 'grey.200',
+                boxShadow: 'none',
+
+                '&:hover': {
+                  borderColor:
+                    theme.palette.mode === 'dark' ? '#173A5E' : 'grey.300',
```

Comment:
For some reason, using `primaryDark`, or even just `primary` was not working. Had to fall back to using plain hex codes 😭

Reply:
That's because the `ExampleCollection.js` is considered a demo so it could not access the branding theme.

To be able to use branding theme, I switch from `demo` to `component` in the markdown file:

```diff
- {{"demo": "ExampleCollection.js", "hideToolbar": true, "bg": "inline"}}
+ {{"component": "docs/src/modules/components/ExampleCollection.js"}}
```

Reply:
Oh, that makes a ton of sense, thank you! Is that documented somewhere? 😅